predict_pv_yield/models/metrics.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+
+def mse_each_forecast_horizon(output: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
```

Comment:
Love these short functions to aid readability of the calling code!

Reply:
thanks